service_status/serverless.yml

```diff
@@ -88,35 +88,7 @@ functions:
     events:
       - schedule:
           rate: rate(3 days)
-          enabled: true
-
-  monitor-service-health:
```

Comment:
This is a junk serverless setting. If not removed, we will end up deploying dead lambda service across the network.